src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -153,338 +152,361 @@ export const FilesAndLicensePanel = ({ uppy }: FilesAndLicensePanelProps) => {
-                        helperText={
-                          associatedLinkHasError
-                            ? (errors.associatedArtifacts?.[associatedLinkIndex] as FormikErrors<AssociatedLink>).id
-                            : null
+                      } else {
+                        const associatedArtifactsBeforeRemoval = associatedArtifacts.length;
+                        remove(associatedLinkIndex);
+                        if (associatedArtifactsBeforeRemoval === 1) {
+                          // Ensure field is set to touched even if it's empty
+                          setFieldTouched(name);
                         }
-                        data-testid={dataTestId.registrationWizard.files.linkToResourceField}
-                        onChange={(event) => {
-                          const inputValue = event.target.value;
-                          if (inputValue) {
-                            if (associatedLinkIndex < 0) {
-                              const newAssociatedLink: AssociatedLink = { type: 'AssociatedLink', id: inputValue };
-                              push(newAssociatedLink);
-                            } else {
-                              const fieldName = `${name}[${associatedLinkIndex}].${SpecificLinkFieldNames.Id}`;
-                              setFieldValue(fieldName, inputValue);
-                              setFieldTouched(fieldName);
-                            }
-                          } else {
-                            const associatedArtifactsBeforeRemoval = associatedArtifacts.length;
-                            remove(associatedLinkIndex);
-                            if (associatedArtifactsBeforeRemoval === 1) {
-                              // Ensure field is set to touched even if it's empty
-                              setFieldTouched(name);
-                            }
-                          }
-                        }}
-                      />
-                    )}
-                  </BackgroundDiv>
-                </Paper>
+                      }
+                    }}
+                  />
+                )}
+              </BackgroundDiv>
+            </Paper>
 
-                {associatedArtifacts.length === 0 &&
-                  typeof errors.associatedArtifacts === 'string' &&
-                  touched.associatedArtifacts && (
-                    <FormHelperText error>
-                      <ErrorMessage name={name} />
-                    </FormHelperText>
-                  )}
+            {(associatedArtifacts.length === 0 || isNullAssociatedArtifact) && !originalDoi && (
+              <Paper elevation={5}>
+                <BackgroundDiv>
+                  <Typography variant="h2" paragraph>
+                    {t('registration.files_and_license.resource_is_a_reference')}
+                  </Typography>
+                  <FormControlLabel
+                    sx={{ backgroundColor: 'white', width: '100%' }}
```

Comment:
Nå er hele hvite raden en checkbox (valgbar), så man kan velge denne med uhell om man trykker helt til høyre som ser ut som bare tomrom. Foreslår noe sånt:
```suggestion
                  <Box sx={{ backgroundColor: 'white', width: '100%', p: '0.25rem 1rem' }}>
                    <FormControlLabel
```
IMO tilfører ikke den hvite bakgrunnen noe annet enn å imitere et tekst-felt den ikke er, så kan godt bare droppes også for min del. Men greit nok om det er noe som er ønsket :)